### PR TITLE
[keycloak] Add Forecastle group configuration

### PR DESCRIPTION
**Pull request by Nuru**
## what
1. [keycloak] Add ability to configure which Forecastle group Keycloak gets added to

## why
1. UI organization

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## what
```
Comment (osterman) [inline docs/pull_request_template.md:1]: Wrong location for this template

Comment (osterman) [reply docs/pull_request_template.md:1]: I guess `docs/` works to, but lets standardize on `.github/PULL_REQUEST_TEMPLATE/
`

**Review by aknysh — APPROVED** [review]

File: releases/keycloak.yaml [hunk 2]
```diff
@@ -75,8 +75,11 @@ releases:
           {{- if eq (env "KEYCLOAK_FORECASTLE_EXPOSE" | default "true") "true" }}
           forecastle.stakater.com/expose: "true"
           forecastle.stakater.com/appName: "Keycloak"
-          {{- if env "FORECASTLE_INSTANCE_NAME" }}
-          forecastle.stakater.com/instance: "{{ env "FORECASTLE_INSTANCE_NAME" }}"
+          {{- if coalesce (env "KEYCLOAK_FORECASTLE_INSTANCE_NAME") (env "FORECASTLE_INSTANCE_NAME") }}
+          forecastle.stakater.com/instance: '{{ coalesce (env "KEYCLOAK_FORECASTLE_INSTANCE_NAME") (env "FORECASTLE_INSTANCE_NAME") }}'
+          {{- end }}
+          {{- if env "KEYCLOAK_FORECASTLE_GROUP" }}
```
Comment (aknysh) [inline releases/keycloak.yaml:81]: @Nuru 
how do we set different `KEYCLOAK_FORECASTLE_GROUP` ENV var for diff sections in `keycloak-gatekeeper-services.yaml`?

Comment (Nuru) [reply releases/keycloak.yaml:81]: This `KEYCLOAK_FORECASTLE_GROUP` variable is for where in the portal the Keycloak Admin UI shows up. Gatekeeper services' groups can be set via `forecastleGroupName` as of `cloudposse/hemlfiles` release 0.55.1

File: docs/pull_request_template.md (hunk 1 above)
Comment (Nuru) [reply docs/pull_request_template.md:1]: If you create a `.github/PULL_REQUEST_TEMPLATE/` directory, then (my understanding is) you no longer have a default template, but instead have to specify which template to use via a query parameter on the pull request URL. I don't like that. 

Really, though, we should be creating [org-wide default templates](https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization) in https://github.com/cloudposse/.github

**Pull request merged**